test/Container/ApplicationFactoryTest.php

```diff
@@ -1296,4 +914,141 @@ public function testProperlyRegistersNestedErrorMiddlewareAsLazyErrorMiddleware(
         $this->assertTrue($r->isClosure(), 'Configured middleware is not the expected lazy-middleware closure');
         $this->assertEquals(4, $r->getNumberOfParameters(), 'Configured middleware is not error middleware');
     }
+
+    /**
+     * @group programmatic
+     */
+    public function testWillNotInjectConfiguredRoutesOrPipelineIfProgrammaticPipelineFlagEnabled()
+    {
+        // @codingStandardsIgnoreStart
```

Comment:
Do we need it? I know that without it `}` must be in new line, but the same we have in line 1036.